src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -62,6 +63,15 @@ public override bool TryMapToProjectedDocumentRange(RazorCodeDocument codeDocume
             }
 
             projectedRange = default;
+
+            if ((originalRange.End.Line < originalRange.Start.Line) ||
```

Comment:
Filed https://github.com/OmniSharp/csharp-language-server-protocol/issues/376 to help simplify this in the future.